src/sophys/common/devices/tatu.py

```diff
@@ -303,3 +301,58 @@ class Tatu9403(TatuBase, CRIO_9403):
             "io23": (TatuOutput, "", {"output_number": "23"}),
         }
     )
+
+
+class Tatu9403V2(TatuBase):
```

Comment:
Shouldn't this follow the same logic as `Tatu9401V2` and subclass from its V1 equivalent?